src/provisioner/provisioner.py

```diff
@@ -118,6 +118,8 @@ def config_apply(solution_config_url: str, cortx_conf_url: str = None,
                     # decoding the byte string in val variable
                     Conf.set(CortxProvisioner._solution_index, key, val.decode('utf-8'))
             CortxProvisioner.apply_cortx_config(cortx_conf, CortxProvisioner.cortx_release)
+            # Adding array count key in conf
+            cortx_conf.add_num_keys()
```

Comment:
Just adding in memory is not enough. We need to store / save it.

Reply:
It is saving in the conf class in utils
 @staticmethod
    def add_num_keys(args):
        """Add "num_xxx" keys for all the list items in ine KV Store"""
        Conf.add_num_keys(ConfCli._index)
        Conf.save(ConfCli._index)